terraform/modules/cloudfront_alb_monitoring/main.tf

```diff
@@ -13,6 +13,7 @@ module "website_cloudfront_alb_monitoring" {
 
   cloudfront_p90_origin_latency_high_alarm_threshold_ms     = 10000
   cloudfront_average_origin_latency_high_alarm_threshold_ms = 6000
+  alb_target_client_error_rate_alarm_threshold_percent      = 10
```

Comment:
Fine by me, but worth considering alternatives:
* Increasing the min traffic that triggers the alarm
* Restricting this to business hours, using the same method as discussed the other day for the ML alarm

Reply:
I'll merge for now, but I've added a note to https://dluhcdigital.atlassian.net/browse/DT-594